app/components/more-details.jsx

```diff
@@ -0,0 +1,92 @@
+// https://github.com/EnCiv/civil-pursuit/issues/89
+
+'use strict'
+import React, { useState } from 'react'
+import Column from './util/column'
+import Submit from './util/submit'
```

Comment:
please make the submit button from buttons.jsx  - the one in utils is for legacy stuff.

please don't use util/column - i don't think you need it and it's also a legacy thing.   Just render within the width of the div.